packages/core/src/App/Components/Layout/Footer/live-chat.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Popover, Icon } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import BinarySocket from '_common/base/socket_base';
+// import { WS } from 'Services';
+import { connect } from 'Stores/connect';
+
+const LiveChat = ({ email, is_logged_in, loginid, landing_company_shortcode, currency, residence }) => {
+    const [is_livechat_interactive, setLiveChatInteractive] = React.useState(false);
+
+    React.useEffect(() => {
+        if (window.LiveChatWidget) {
+            BinarySocket.wait('get_settings').then(response => {
+                const get_settings = response.get_settings || {};
+                const { first_name, last_name } = get_settings;
+
+                if (email) window.LiveChatWidget.call('set_customer_email', email);
+                if (first_name && last_name)
+                    window.LiveChatWidget.call('set_customer_name', `${first_name} ${last_name}`);
+            });
+
+            window.LiveChatWidget.on('ready', () => {
+                setLiveChatInteractive(true);
+            });
+            window.LiveChatWidget.on('visibility_changed', ({ visibility }) => {
+                // only visible to CS
+                let session_variables = { loginid: '', landing_company_shortcode: '', currency: '', residence: '' };
+
+                if (visibility === 'maximized' && is_logged_in) {
+                    session_variables = {
+                        ...(loginid && { loginid }),
+                        ...(landing_company_shortcode && { landing_company_shortcode }),
+                        ...(currency && { currency }),
+                        ...(residence && { residence }),
+                    };
+
+                    window.LiveChatWidget.call('set_session_variables', session_variables);
+                }
+
+                if (visibility === 'maximized' && !is_logged_in) {
+                    window.LiveChatWidget.call('set_customer_email', ' ');
+                    window.LiveChatWidget.call('set_customer_name', ' ');
```

Comment:
let's make it empty instead of an extra space?

Reply:
for some reason, the call does not accept empty string, so i checked in binary static, its adding space in the string, so i matched the implementation with binary static

Reply:
@negar-binary the LiveChatWidget will throw an error for empty strings - so to e.g. reset the name you have to pass a empty space